workflows/diagnostics/fv3net/diagnostics/prognostic_run/compute.py

```diff
@@ -146,34 +162,61 @@ def _is_empty(data: Union[xr.Dataset, xr.DataArray]) -> bool:
 def ignore_nan_values(x, y, w):
     if "time" not in w.dims and "time" in x.dims:
         w = w.expand_dims(dim={"time": x.time})
-    mask = np.isnan(x) | np.isnan(y)
-    x = x.where(~mask, drop=True)
-    y = y.where(~mask, drop=True)
-    w = w.where(~mask, drop=True)
+    mask = (x.notnull() | y.notnull()).compute()
+    x = x.where(mask, drop=True)
+    y = y.where(mask, drop=True)
+    w = w.where(mask, drop=True)
     return x, y, w
 
 
 def rms(x, y, w, dims):
     with xr.set_options(keep_attrs=True):
-        if np.isnan(x).any() or np.isnan(y).any():
-            x, y, w = ignore_nan_values(x, y, w)
-        return np.sqrt(((x - y) ** 2 * w).sum(dims) / w.sum(dims))
+        return xr.ufuncs.sqrt(((x - y) ** 2 * w).sum(dims) / w.sum(dims))
```

Comment:
It looks like xarray has dropped all ufuncs in all favor of numpy’s ufuncs after 2022.06. So I think numpy sqrt would be the same already. Does this make much difference here performance wise?

Reply:
I believe I actually got some kind of error here or was debugging when I made the switch.  Or it could've been something related to me trying to remove explicit numpy references.  I'm fine switching back to see if it works.

Reply:
I'll try a run with np.sqrt and see if it makes a big difference. I would advocate for sticking with numpy in prep for the day when we have to upgrade xarray and everything breaks...

Reply:
New run with np.sqrt did not show obvious slowdown. I would opt for keeping the original form. Also, we should keep the ignore nan check because it would nan out variables like T850 where there're actual nans.

Reply:
I switched it back to including the NaN masking (without drop since xarray sum will ignore NaN) and using `np.sqrt`